Update CONTRIBUTING.md (#51728)
dotnet/aspnetcore: CONTRIBUTING.md

  Over the years we've seen many PRs targeting areas of the framework, which we didn't plan to expand further at the time.
  In all these cases we had to say `no` to those PRs and close them. That, obviously, is not a great outcome for us. And it's especially bad for the contributor, as they've spent a lot of effort preparing the change.

>   To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, look for a comment from our team, which follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as procvides references and hits about how to approach the problem at hand.

Reviewer: ```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.
```

Reviewer: I assume we won't be able to annotate all the existing help wanted (?)

Reviewer 2: > I assume we won't be able to annotate all the existing help wanted (?)

Yep, there's a few issues that already have summaries that need to be updated to follow the format but we'll need to make concerted effort to update the rest.

>   To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, look for a comment from our team, which follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as procvides references and hits about how to approach the problem at hand.

Reviewer 3: ```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, look for a comment from our team, which follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is meant to make it much simpler for you to both understand what the problem is as well as procvides references and hits about how to approach the problem at hand.
```

>   To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is menat to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.

Author: ```suggestion
  To resolve this problem, we've decided to separate a bucket of issues, which would be great candidates for community members to contribute to. We mark these issues with the `help wanted` label. You can find all these issues at [https://aka.ms/aspnet/helpwanted](https://aka.ms/aspnet/helpwanted). When you choose an issue you'd like to work on, you'll see that some include a comment from our team that follows the [Issue Summary Template](/docs/HelpWantedIssueSummaryCommentTemplate.md). That comment is meant to make it much simpler for you to both understand what the problem is as well as provide references and hints about how to approach it.
```
